src/graph/query_graph.c

```diff
@@ -171,11 +173,11 @@ static void _QueryGraph_ExtractNode
 
 			QueryGraph_AddNode(graph, n);
 			// set node label information
-			_QueryGraphSetNodeLabel(n, ast_node);
+			_QueryGraphSetNodeLabel(n, ast_node, false);
```

Comment:
add comment explain how we get to this

Reply:
Reached when a node is first met in the extraction algorithm. Also, I can add a comment of this if you think it's relevant, but it's quite clear from the code.